config/locales/views/admin/en.log_events.yml

```diff
@@ -5,6 +5,8 @@ en:
         dropdown_titles:
           order_detail:
             problem_queue: Order added to problem queue
+          user:
+            internal_changed: User price group changed
```

Comment:
I'm not sure I like `internal_changed`. It seems like it's not quite descriptive enough. Maybe `default_price_group_changed`, or `base_price_group_changed`?